server/honeybadger/views.py

```diff
@@ -270,7 +271,8 @@ def api_beacon(target, agent):
         logger.error('Invalid target GUID.')
         abort(404)
     # extract universal parameters
-    comment = request.values.get('comment', '').decode('base64') or None
+    #comment = request.values.get('comment', '').decode('base64') or None
```

Comment:
Can probably just delete this now, right?

Reply:
Yes, I had forgotten about this comment. I'll remove it.

Reply:
👍

Reply:
This is done.